src/Akeneo/Pim/Structure/Bundle/Resources/workspaces/settings-ui/src/components/datagrids/attribute-groups/AttributeGroupsDataGrid.tsx

```diff
@@ -38,11 +39,11 @@ const AttributeGroupsDataGrid: FC<Props> = ({groups, onGroupCountChange}) => {
 
   return (
     <>
-      <SearchBar
-        count={filteredGroups.length}
-        searchValue={searchString === undefined ? '' : searchString}
-        onSearchChange={onSearch}
-      />
+      <Search searchValue={searchString === undefined ? '' : searchString} onSearchChange={onSearch}>
```

Comment:
```suggestion
      <Search searchValue={searchString} onSearchChange={onSearch}>
```
I don't see how `searchString` could be `undefined`